FortranCon2020-community/README.md

```diff
@@ -4,4 +4,23 @@
 
 **Title**: Toward a thriving open source Fortran community
 
-**Abstract**: TODO
+**Abstract**:
+A thriving community around a programming language is essential for onboarding
+newcomers to the language and retaining existing users.
+Besides a few discussion boards and mailing lists, Fortran has not had a 
+healthy online community in the modern internet era, like many other 
+programming languages have.
+Despite the developments of the language itself, its user base has been 
+declining as a result. 
+Many new software projects are now being started in languages like C++, Python,
+or Julia, despite Fortran being better suited for the task at hand.
+In this talk, we'll discuss the need for and the ongoing development of the new
```

Comment:
```suggestion
In this talk, we will discuss the need for and the ongoing development of the new
```
I think it's better to use the expanded form in writing.

Reply:
It's a matter of style and I prefer contractions where convenient as they allow for a personal and truer to self voice. I accept this suggestion here because this is community work.